transformations/chinese_simplified_traditional_perturbation/transformation.py

```diff
@@ -0,0 +1,77 @@
+import itertools
+import random
+import opencc
+from interfaces.SentenceOperation import SentenceOperation
+from tasks.TaskTypes import TaskType
+
+
+
+def chinese_simplified_traditional_perturbation(text,
+                          prob,
+                          seed,
+                          max_outputs,
+                          config
+                          ):
+    random.seed(seed)
+
+    perturbed_texts = []
+
+    if(config == 's2t'):
+        converter = opencc.OpenCC('s2t.json')
+    elif (config == 't2s'):
+        converter = opencc.OpenCC('t2s.json')
+    else:
+        print("Configuration not specified")
+    for _ in itertools.repeat(None, max_outputs):
```

Comment:
You could simplify by removing the import of itertools and using range: `for _ in range(max_outputs):`

Reply:
Addressed